fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -465,13 +465,13 @@ public void addTokenValuesView(List<IDatabaseObject> group, String prefix) {
             // in the join condition to give the optimizer the best chance at finding a good nested
             // loop strategy
             select.append("SELECT ref.").append(MT_ID);
-            select.append(", ref.parameter_name_id, ctv.code_system_id, ctv.token_value, ref.logical_resource_id ");
+            select.append(", ref.parameter_name_id, ctv.code_system_id, ctv.token_value, ref.logical_resource_id, ref.ref_version_id ");
             select.append(" FROM ").append(commonTokenValues.getName()).append(" AS ctv, ");
             select.append(resourceTokenRefs.getName()).append(" AS ref ");
             select.append(" WHERE ctv.common_token_value_id = ref.common_token_value_id ");
             select.append("   AND ctv.").append(MT_ID).append(" = ").append("ref.").append(MT_ID);
         } else {
-            select.append("SELECT ref.parameter_name_id, ctv.code_system_id, ctv.token_value, ref.logical_resource_id ");
+            select.append("SELECT ref.parameter_name_id, ctv.code_system_id, ctv.token_value, ref.logical_resource_id, ref.ref_version_id ");
```

Comment:
@punktilious Is adding the ref_version_id column to the xxx_TOKEN_VALUES_V view considered a schema change, and if so, is there anything special we need to do?

Reply:
Will let robin reply to this, but that view was added some time ago...I don't think there's anything we need to do as part of this PR other than just use it

Reply:
Yes, it's a schema change. You should create a new entry in the FhirSchemaVersion enum (which I think will be V0007. You can then use V0007.vid() as the version id value for the view: 
```
        View view = View.builder(schemaName, viewName)
                .setVersion(FhirSchemaVersion.V0007.vid())
                .setSelectClause(select.toString())
                .addPrivileges(resourceTablePrivileges)
                .addDependency(commonTokenValues)
                .addDependency(resourceTokenRefs)
                .build();
```
This will make sure it's applied when the schema update tool is run.

Reply:
Any circumstance where this may not work in migration?  e.g. is CreateOrReplaceViewDAO resilient to signature or *View* changes from v1 to v2. And I think with the version not set, we'd not get a new view created.

Reply:
Ah, makes sense.  I saw the SELECTs and got confused.  I retract my comment :-)

Reply:
new schema version created and view updated to point to it